src/kili/mutations/asset/helpers.py

```diff
@@ -31,3 +33,26 @@ def process_update_properties_in_assets_parameters(properties) -> Dict:
         map(is_none_or_empty, properties["to_be_labeled_by_array"])
     )
     return properties
+
+
+def get_asset_ids_or_throw_error(
+    kili,
+    asset_ids: Optional[List[str]] = None,
+    external_ids: Optional[List[str]] = None,
+    project_id: Optional[str] = None,
+) -> List[str]:
+    """
+    Check if external id to internal id is valid and needed.
+    """
+    if asset_ids is None and external_ids is None:
```

Comment:
for all of this code before the `infer_ids_from_external_ids`, I have already coded a function `check_asset_identifier_arguments` in `mutations/label/helpers`.

Reply:
thanks! I move the helpers.py to mutations folder